CMakeLists.txt

```diff
@@ -124,7 +124,7 @@ function(relatedrepo_GetClosestMatch)
 
   # Otherwise, try to use a branch that matches `current_head` in the fork repository
   execute_process(COMMAND ${GIT_EXECUTABLE} ls-remote --heads --tags
-                  ${__ORIGIN_PREFIX}/${__REPO_NAME} ${current_head} OUTPUT_VARIABLE matching_refs)
+                  ${__ORIGIN_PREFIX}/${__REPO_NAME} ${current_head} OUTPUT_VARIABLE matching_refs ERROR_QUIET)
```

Comment:
Is the `ERROR_QUIET` still needed?

Reply:
> Is the `ERROR_QUIET` still needed?

I find this more confusing than helpful:
```diff
  -- 193/247 python tutorials have been activated.
  -- Searching for "origin" repo of roottest: error: No such remote 'origin'
+
+ fatal: '/roottest' does not appear to be a git repository
+ fatal: Could not read from remote repository.
+ 
+ Please make sure you have the correct access rights
+ and the repository exists.
  -- Found roottest: /Users/shageboeck/code/root-src/../roottest
```

Note especially (last line) how it does actually find roottest, and how ROOT keeps configuring correctly, albeit with whatever the current branch in roottest is.